src/arrays.js

```diff
@@ -1,61 +1,73 @@
 const getNthElement = (index, array) => {
-  // your code here
+  
+  if (index >= array.length){
+    return array[index-array.length];
+  }
+  return array[index];
 };
 
 const arrayToCSVString = array => {
-  // your code here
+const csv = array.toString();
+return(csv);
 };
 
 const csvStringToArray = string => {
-  // your code here
+const array = string.split(',');
+return(array);
 };
 
 const addToArray = (element, array) => {
-  // your code here
+array.push(element);
 };
 
 const addToArray2 = (element, array) => {
-  // your code here
+return array.concat(element);
 };
 
 const removeNthElement = (index, array) => {
-  // your code here
+array.splice(index, 1);
 };
 
 const numbersToStrings = numbers => {
-  // your code here
+return numbers.map(String);
 };
 
 const uppercaseWordsInArray = strings => {
-  // your code here
+
 };
 
 const reverseWordsInArray = strings => {
-  // your code here
+
 };
 
 const onlyEven = numbers => {
-  // your code here
+  const even = numbers.filter((num) => num % 2 === 0);
+  return even;
 };
 
 const removeNthElement2 = (index, array) => {
-  // your code here
+return array.splice(index, 2,0);
 };
 
 const elementsStartingWithAVowel = strings => {
-  // your code here
+const vowels = ['a', 'e', 'i', 'o', 'u'];
+const elementsWithVowel = strings.filter((string) => string.startsWith(vowels/i));
+return elementsWithVowel;
```

Comment:
It looks to me like you're trying to use regex (based on the /i) but the `vowels` variables is an array (not regex). Furthermore, `startsWith` only accepts a string as a parameter. To tackle this problem using regex you do something like:
```
const elementsStartingWithAVowel = strings => {
  return strings.filter(string => string[0].match(/[aeiou]/i));
};
```
Here, in my filter, I have used `string[0]` to only test the first letter of the string. I then use `.match()` to test that letter against some regex. The regex I use is `/[aeiou]/i` which specifies it must be only one of those letters. The `i` after the slash dictates that this is case insensitive.